File: jambeth/jambeth-security/src/main/java/de/osthus/ambeth/privilege/transfer/IPropertyPrivilegeOfService.java
package de.osthus.ambeth.privilege.transfer;

import de.osthus.ambeth.annotation.XmlType;

@XmlType
public interface IPropertyPrivilegeOfService
{
	boolean isCreateAllowed();

	boolean isReadAllowed();

	boolean isUpdateAllowed();

	boolean isDeleteAllowed();
}

File: ambeth/Ambeth.Security/ambeth/privilege/transfer/IPropertyPrivilegeOfService.cs
using De.Osthus.Ambeth.Annotation;

namespace De.Osthus.Ambeth.Privilege.Transfer
{
    [XmlType]
    public interface IPropertyPrivilegeOfService
    {
        bool ReadAllowed { get; }

        bool CreateAllowed { get; }

        bool UpdateAllowed { get; }

        bool DeleteAllowed { get; }
    }
}